src/runtime/offers-flow-test.js

```diff
@@ -193,3 +199,108 @@ describes.realWin('SubscribeOptionFlow', {}, env => {
     });
   });
 });
+
+describes.realWin('Abbrv Offer flow', {}, env => {
```

Comment:
`'AbbrvOfferFlow'` - b/c this is the most likely thing someone would search the tests for `AbbrvOfferFlow` class.

Reply:
Done